deployments/fuji/usdc/migrations/1665028496_absorb_transfer_event.ts

```diff
@@ -0,0 +1,62 @@
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, proposal } from '../../../../src/deploy';
+
+import { expect } from 'chai';
+
+export default migration('1665028496_absorb_transfer_event', {
+  async prepare(deploymentManager: DeploymentManager) {
+    const cometFactory = await deploymentManager.deploy('cometFactory', 'CometFactory.sol', [], true);
+    return { newFactoryAddress: cometFactory.address };
+  },
+
+  async enact(deploymentManager: DeploymentManager, { newFactoryAddress }) {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+
+    const {
+      governor,
+      comet,
+      configurator,
+      cometAdmin,
+      COMP,
+    } = await deploymentManager.getContracts();
+
+    const actions = [
+      // 1. Set comet factory to newly deployed factory
+      {
+        contract: configurator,
+        signature: 'setFactory(address,address)',
+        args: [comet.address, newFactoryAddress],
+      },
+
+      // 2. Deploy and upgrade to a new version of Comet
+      {
+        contract: cometAdmin,
+        signature: 'deployAndUpgradeTo(address,address)',
+        args: [configurator.address, comet.address],
+      },
+    ];
+    const description = "# Liquidation Event Handling\n";
+    const txn = await deploymentManager.retry(
+      async () => governor.propose(...await proposal(actions, description))
+    );
+    trace(txn);
+
+    const event = (await txn.wait()).events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async verify(deploymentManager: DeploymentManager) {
+    const {
+      governor,
+      comet,
+      configurator,
+      rewards,
+      COMP,
+    } = await deploymentManager.getContracts();
+
+    // 1. & 2.
+    //  added a scenario to check for new Transfer event
```

Comment:
is this a todo?

Reply:
No its there

Reply:
is there a reason to call `getContracts()` then?

Reply:
I think I was going to check that the address had changed, but I removed this and updated the scen name / description